core/akka-runtime/src/main/scala/com/ing/baker/runtime/RecipeManager.scala

```diff
@@ -0,0 +1,13 @@
+package com.ing.baker.runtime
+
+import com.ing.baker.il.CompiledRecipe
+
+import scala.concurrent.Future
+
+trait RecipeManager {
+  def add(compiledRecipe: CompiledRecipe): Future[String]
```

Comment:
maybe there can be some better name for this method, like `put()` or `upsert()` ? 
- seems like it has 'insert or update if exists' semantics